src/PerformanceTests/VisualBasic/Program.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Running;
+
+namespace PerformanceTests
+{
+    public class Program
```

Comment:
😕 Does this _need_ to be a separate project?

Reply:
nope, these could be combined.